src/main/resources/com/google/api/codegen/csharp/discovery_fragment.snip

```diff
@@ -219,16 +219,10 @@
   @end
 @end
 
-# Unlike placeholders for other languages, this placeholder does not pretty-print output.
-# This seems to require a third-party library like Json.NET: https://www.nuget.org/packages/newtonsoft.json/
 @private print(element)
-    @let JavaScriptSerializer = context.using("System.Web.Script.Serialization.JavaScriptSerializer")
-        Console.WriteLine(new {@JavaScriptSerializer}().Serialize({@element}));
-    @end
+    Console.WriteLine({@element});
 @end
 
 @private printEntry(entry)
-    @let JavaScriptSerializer = context.using("System.Web.Script.Serialization.JavaScriptSerializer")
-        Console.WriteLine("[" + {@entry}.Key + "] = " + new {@JavaScriptSerializer}().Serialize({@entry}.Value));
-    @end
+    Console.WriteLine("[" + {@entry}.Key + "] = " + {@entry}.Value);
```

Comment:
Do we not have a baseline that exercises this codepath? If we don't, could you please start and share a document with baseline features we need to test and include "map response" in it? That way when we do the synthetic (manual or autogen) API/baseline, we make sure this is among the cases covered.